orthconfig-template

```diff
@@ -77,10 +77,13 @@ ort_docker_registry_password=""
 ort_docker_image="registry.gitlab.com/oss-review-toolkit/ort-gitlab-ci/ort:latest"
 
 #
-# Artifactory credentials.
+# netrc file credentials to inject when running ORT docker.
 #
-artifactory_user_name=""
-artifactory_password=""
+# Note that the Pip package manager uses the netrc file for authenticating against artifact repositories.
```

Comment:
It's also used by Git and NPM, and probably more package managers (maybe @sschuberth knows which if you want to list them here).